os/temp/temp_dir.go

```diff
@@ -75,3 +77,13 @@ func TempDirHere() (*TempDir, error) {
 
 	return NewTempDir(cwd, "scoot-tmp-")
 }
+
```

Comment:
Best judgement I guess, when something is too big it has to be split out, this is tiny though.